curriculum/js1/3.js

```diff
@@ -5,8 +5,16 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (a, b = 'hello') => {
```

Comment:
If you start your base case `b` with empty string, then you don't need 2 if statements